client/src/components/hero/Hero.tsx

```diff
@@ -17,64 +20,178 @@ interface HomeProps {
+  // created this for the case when the sub box needs to show the first announcement and the main box shows last announcement within the list of announcements
+  const [subAnnouncementPosition, setSubAnnouncementPosition] = useState(1);
+
+  const history = useHistory();
+
+  let announcementLength = 0;
+
+  const incrementAnnouncement = () => {
+    {
+      mainAnnouncementPosition >= announcementLength
+        ? setMainAnnouncementPosition(0)
+        : setMainAnnouncementPosition(mainAnnouncementPosition + 1);
+    }
+
+    {
+      subAnnouncementPosition >= announcementLength
+        ? setSubAnnouncementPosition(0)
+        : setSubAnnouncementPosition(subAnnouncementPosition + 1);
+    }
+  };
+
+  const decrementAnnouncement = () => {
+    {
+      mainAnnouncementPosition === 0
+        ? setMainAnnouncementPosition(announcementLength)
+        : setMainAnnouncementPosition(mainAnnouncementPosition - 1);
+    }
+    {
+      subAnnouncementPosition === 0
+        ? setSubAnnouncementPosition(announcementLength)
+        : setSubAnnouncementPosition(subAnnouncementPosition - 1);
+    }
+  };
+
+  if (!viewer?.announcements) {
+    return <div>loading...</div>;
+  } else {
+    announcementLength = viewer.announcements.length - 1;
+  }
+
   return (
     <Box className={className}>
       <Box className="hero" display="flex" alignItems="center">
-        <Container>
-          <Box display="flex" justifyContent="flex-start" alignItems="center">
-            {/* Main image */}
-            <Box>
-              <img src={HeroMainImage} alt="hero-main" className="hero-main" />
-            </Box>
+        {viewer.announcements.length > 0 && (
+          <Container>
+            <ChevronLeftIcon
+              className="chevron-left"
+              onClick={() => decrementAnnouncement()}
+            />
+            <Box display="flex" justifyContent="center" alignItems="center">
+              {/* Main image */}
 
-            {/* Sub section */}
-            <Box
-              display="flex"
-              flexDirection="column"
-              justifyContent="space-between"
-              alignItems="left"
-              height={385}
-              ml={2}
-            >
-              <Box className="hero-sub">
+              <motion.div
+                initial={{ opacity: 0, x: 100, y: 0 }}
+                animate={{ opacity: 1, x: 0, y: 0 }}
+                transition={{ delay: 0.5 }}
+              >
                 <Box display="flex" justifyContent="flex-start">
-                  <Box>
-                    <img src={HeroSubImage} alt="hero-sub" className="hero-sub" />
-                  </Box>
+                  <img
+                    src={
+                      viewer.announcements[mainAnnouncementPosition].imageURL
+                    }
+                    alt="hero-main"
+                    className="hero-main-image"
+                  />
+                  <VerticalDivider
```

Comment:
new line - try to put new line between each components or sections. Makes code more readable.